src/components/progress/progress.js

```diff
@@ -28,8 +28,8 @@ Progress.propTypes = {
   style: PropTypes.shape({}),
   color: PropTypes.oneOf(colors),
   size: PropTypes.oneOf(['small', 'medium', 'large']),
-  value: PropTypes.number.isRequired,
-  max: PropTypes.number.isRequired,
```

Comment:
Thank for the PR, can you add default value of undefined to this two props? Thanks

Reply:
Sure thing.